src/Umbraco.Infrastructure/Persistence/Dtos/ContentVersionCultureVariationDto.cs

```diff
@@ -19,6 +19,7 @@ internal class ContentVersionCultureVariationDto
     [Column("versionId")]
     [ForeignKey(typeof(ContentVersionDto))]
     [Index(IndexTypes.UniqueNonClustered, Name = "IX_" + TableName + "_VersionId", ForColumns = "versionId,languageId")]
+    [Index(IndexTypes.NonClustered, Name = "IX_" + TableName, ForColumns = "versionId", IncludeColumns = "id,languageId,name,date,availableUserId")]
```

Comment:
The existing index could be used here if the columns are included

Reply:
@nzdev , same question as for previous comment: which name should be used for index?

Reply:
Original.